qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/web/views/SearchCustomerView.groovy

```diff
@@ -0,0 +1,256 @@
+package life.qbic.portal.qoffer2.web.views
+
+import com.vaadin.data.ValidationException
+import com.vaadin.data.ValidationResult
+import com.vaadin.data.Validator
+import com.vaadin.data.ValueContext
+import com.vaadin.icons.VaadinIcons
+import com.vaadin.server.UserError
+import com.vaadin.ui.Alignment
+import com.vaadin.ui.Button
+import com.vaadin.ui.FormLayout
+import com.vaadin.ui.Grid
+import com.vaadin.ui.HorizontalLayout
+import com.vaadin.ui.TextField
+import groovy.util.logging.Log4j2
+import life.qbic.datamodel.dtos.business.Customer
+import life.qbic.portal.qoffer2.web.viewmodel.SearchCustomerViewModel
+import life.qbic.portal.qoffer2.web.viewmodel.ViewModel
+
+/**
+ * This class generates a Form Layout with which the user can search if a customer is already contained in the database
+ *
+ * SearchCustomerView will be integrated into the qOffer 2.0 Portlet and provides an User Interface
+ * with the intention of enabling a user the search of a Customer by looking for the customer's first and lastname to the QBiC Database
+ *
+ * @since: 1.0.0*
+ */
+@Log4j2
+class SearchCustomerView extends FormLayout {
+
+    final private ViewModel viewModel
+    final private SearchCustomerViewModel searchCustomerViewModel
+    private TextField firstNameField
+    private TextField lastNameField
+    private Button submitButton
+    private Button clearButton
+    public Grid<Customer> customerGrid
+    public HorizontalLayout gridLayout
+    private List<Customer> foundCustomerList
+
+    private Boolean firstNameValid
+    private Boolean lastNameValid
```

Comment:
what is valid? Maybe be more explicit with the naming to expose the function of this variable. What about:
```suggestion
    private Boolean firstNameSet
    private Boolean lastNameSet
```
or 
```suggestion
    private Boolean firstNameProvided
    private Boolean lastNameProvided
```